Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
Yes, that's exactly what I mean.

Where if FeatureName is not filled out, use the resource name instead. So allow for it to be passed as well